include/seqan3/alignment/pairwise/alignment_result.hpp

```diff
@@ -200,6 +209,43 @@ class alignment_result
         return data.alignment;
     }
     //!\}
+
+    //!\cond DEV
```

Comment:
Why is this wrapped in DEV when the class is part of detail anyway?

Reply:
No the class itself is not in detail. Only the internal value type.